modules/infra/digitalocean/main.tf

```diff
@@ -49,6 +49,47 @@ resource "digitalocean_droplet" "droplet" {
   }
 }
 
+resource "digitalocean_loadbalancer" "k8s_api_loadbalancer" {
+  name   = "${var.prefix}-6443-lb"
+  region = var.region
+  forwarding_rule {
+    entry_port     = 443
+    entry_protocol = "https"
+
+    target_port     = 6443
+    target_protocol = "https"
+
+    tls_passthrough = true
+  }
+  healthcheck {
+    port     = 6443
+    protocol = "tcp"
+  }
+
+  droplet_ids = resource.digitalocean_droplet.droplet[*].id
+  depends_on  = [digitalocean_droplet.droplet]
+}
+
+resource "digitalocean_loadbalancer" "https_loadbalancer" {
```

Comment:
Same as above.

Reply:
Made these changes in https://github.com/rancher/tf-rancher-up/pull/147/commits/31ea706bcaad436b1d3be2c0e063c14f98ff39f3

Reply:
Handled this in https://github.com/rancher/tf-rancher-up/pull/147/commits/31ea706bcaad436b1d3be2c0e063c14f98ff39f3